scss/_layouts_application.scss

```diff
@@ -0,0 +1,170 @@
+@import 'layouts_application-panels';
+
+@mixin vf-l-application {
+  .l-application {
+    display: grid;
+    grid-template-areas:
+      'nav  navbar navbar'
+      'nav  main   aside'
+      'nav  status status';
+    grid-template-columns: min-content 1fr min-content;
+    grid-template-rows: min-content 1fr min-content;
+    height: 100vh;
+    width: 100vw;
+  }
+
+  $panel-drop-shadow: 0 0 2rem 0 rgba($color-x-dark, $shadow-opacity);
+  $panel-drop-shadow-transparent: 0 0 0 0 transparent;
+
+  // Top navigation bar to toggle side navigation on small screens
+  .l-navigation-bar {
+    grid-area: navbar;
+  }
+
+  // Navigation panel/drawer
+  .l-navigation {
+    bottom: 0;
+    box-shadow: $panel-drop-shadow;
+    height: 100vh;
+    left: 0;
+    overflow-y: auto;
+    position: fixed;
+    top: 0;
+    width: 100%;
+    z-index: 103; // consistent with side-navigation component, above global nav
+
+    @media (min-width: $breakpoint-x-small) {
+      width: auto;
+    }
+
+    &.is-collapsed {
+      width: 0;
+    }
+  }
+
+  $layout-navigation-collapsed-width: 3rem;
+  $layout-navigation-expanded-width: 15rem;
+
+  .l-navigation__drawer {
+    height: 100vh;
+    width: auto;
+
+    @media (min-width: $breakpoint-x-small) {
+      width: $layout-navigation-expanded-width;
+    }
+  }
+
+  @media (min-width: $breakpoint-medium) {
+    .l-navigation-bar {
+      // use navigation bar as a grid spacer for collapsed navigation
+      grid-area: nav;
+      overflow: hidden;
+      width: $layout-navigation-collapsed-width;
+    }
+
+    .l-navigation {
+      box-shadow: $panel-drop-shadow-transparent;
+      transition: width 100ms, box-shadow 100ms;
```

Comment:
Can you use the animation settings for consistency

Reply:
Sure, I always forget that `vf-animation` exists and is meant for transitions...